var/spack/repos/builtin/packages/git/package.py

```diff
@@ -57,6 +57,13 @@ class Git(Package):
     # version('2.5.4', '3eca2390cf1fa698b48e2a233563a76b')
     # version('2.2.1', 'ff41fdb094eed1ec430aed8ee9b9849c')
 
+    resource(
+        name="git-manpages",
+        url="https://www.kernel.org/pub/software/scm/git/"
+        "git-manpages-2.11.0.tar.xz",
```

Comment:
but it is **clearly incorrect** to use this resource regardless of the git version being installed.